apps/src/templates/sectionAssessments/StudentAssessmentOverviewTable.jsx

```diff
@@ -0,0 +1,174 @@
+        sortingOrder: {
+          FIRST: 'asc',
+          asc: 'desc',
+          desc: 'asc'
+        },
+        selectedColumn
+      })
+    });
+  };
+
+  correctAnswerColumnFormatter = (answers, {rowData, columnIndex}) => {
+    const cell = rowData.answers;
+
+    let answerCell = '';
+      cell.forEach((answer) => {
+        if (answer.isCorrectAnswer) {
+          answerCell = answer.multipleChoiceOption;
+        }
+      });
+
+    return (
+      <MultipleChoiceAnswerCell
+        id={rowData.id}
+        answerCell={answerCell}
+      />
+    );
+  };
+
+  studentAnswerColumnFormatter = (studentAnswers, {rowData, rowIndex}) => {
+    let cell = '';
+    const selectStudentAnswers = this.props.studentAnswerData[0].studentAnswers[rowIndex];
+    selectStudentAnswers.answer.forEach((answerSelection) => {
+      cell = answerSelection;
+    });
+
+    let answerCell = '';
+    rowData.answers.forEach ((answer) => {
+      if (answer.isCorrectAnswer) {
+        answerCell = answer.multipleChoiceOption;
+      }
+    });
+
+    return (
+      <MultipleChoiceAnswerCell
+        id={rowData.id}
+        answerCell={cell}
+        isCorrectAnswer={(cell === answerCell)}
+      />
+    );
+  };
+
+  getColumns = (sortable) => {
+    let dataColumns = [
+      {
+        property: 'question',
+        header: {
+          label: commonMsg.question(),
+          props: {style: tableLayoutStyles.headerCell},
+          transforms: [sortable],
+        },
+        cell: {
+          props: {style: tableLayoutStyles.cell},
+        }
+      },
+      {
+        property: 'studentAnswer',
+        header: {
+          label: commonMsg.studentAnswer(),
+          props: {style: tableLayoutStyles.headerCell},
+        },
+        cell: {
+          format: this.studentAnswerColumnFormatter,
+          props: {style: tableLayoutStyles.cell},
+        }
+      },
+      {
+        property: 'correctAnswer',
+        header: {
+          label: commonMsg.checkCorrectAnswer(),
+          props: {style: tableLayoutStyles.headerCell},
+        },
+        cell: {
+          format: this.correctAnswerColumnFormatter,
+          props: {style: tableLayoutStyles.cell},
+        }
+      },
+    ];
+    return dataColumns;
+  };
+
+  render() {
+    // Define a sorting transform that can be applied to each column
+    const sortable = wrappedSortable(this.getSortingColumns, this.onSort, sortableOptions);
+    const columns = this.getColumns(sortable);
+    const sortingColumns = this.getSortingColumns();
+
+    const sortedRows = sort.sorter({
+      columns,
+      sortingColumns,
+      sort: orderBy,
+    })(this.props.questionAnswerData);
+
+    return (
+      <Table.Provider
+        columns={columns}
+        style={tableLayoutStyles.table}
+      >
+        <Table.Header />
+        <Table.Body rows={sortedRows} rowKey="id" />
+      </Table.Provider>
+    );
+  }
+}
+
+export default StudentAssessmentOverviewTable;
+
```

Comment:
nit: only need one extra space at the end of a file

Reply:
(This is on the list of thing's I'd definitely turn on linting for when we have seamless autofix working...)